### PR TITLE
CASMINST-5746

### PR DESCRIPTION
**Pull request by atifsyedali**
## Summary and Scope

IUF operator should conditionally include operations based IUF product manifest contents

_Is this change backwards incompatible, backwards compatible, or a backwards compatible bugfix?_

Yes

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [CASMINST-5746](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-5746)

## Testing

_List the environments in which these changes were tested._

frigg

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: charts/v1.0/cray-nls/Chart.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 apiVersion: v2
 name: "cray-nls"
-version: 1.4.48
+version: 1.4.50-debug5
```
Comment (haasken-hpe) [inline charts/v1.0/cray-nls/Chart.yaml:27]: Should we even be doing this here anymore? Is it just for your testing? I'd really like to remove the charts that exist in two repos ASAP.

File: charts/v1.0/cray-nls/values.yaml [hunk 2]
```diff
@@ -104,7 +104,7 @@ cray-service:
       name: "cray-nls"
       image:
         repository: artifactory.algol60.net/csm-docker/stable/cray-nls
-        tag: 0.9.17
+        tag: 0.9.18-debug5
```
Comment (haasken-hpe) [inline charts/v1.0/cray-nls/values.yaml:107]: I assume if we are going to touch the charts here, the version here should at least match the 0.9.21 above.

**Review by haasken-hpe — APPROVED** [review]

File: charts/v1.0/cray-nls/Chart.yaml (hunk 1 above)
Comment (atifsyedali) [reply charts/v1.0/cray-nls/Chart.yaml:27]: It is just for my testing.

File: charts/v1.0/cray-nls/values.yaml (hunk 2 above)
Comment (atifsyedali) [reply charts/v1.0/cray-nls/values.yaml:107]: yep https://github.com/Cray-HPE/cray-nls-charts/pull/26/files

**Pull request merged**